client/src/banners/active-banner.tsx

```diff
@@ -71,24 +71,25 @@ function SendCTAEventToGA(eventCategory: string) {
   });
 }
 
-function RedesignAnnouncementBanner({
+function PlusLaunchAnnouncementBanner({
   onDismissed,
 }: {
   onDismissed: () => void;
 }) {
   return (
-    <Banner id={REDESIGN_ANNOUNCEMENT} onDismissed={onDismissed}>
+    <Banner id={PLUS_LAUNCH_ANNOUNCEMENT} onDismissed={onDismissed}>
```

Comment:
Shouldn't this banner only show if `PLUS_IS_ENABLED` **and** available in the user's region (or do we only know the user's region once they're logged in)?

```tsx
    {PLUS_IS_ENABLED && (<Banner id={PLUS_LAUNCH_ANNOUNCEMENT} onDismissed={onDismissed}>
```

Reply:
> or do we only know the user's region once they're logged in

Oh dang, that is a good question. I have a feeling we probably only know that after they are logged in. @Guyzeroth do you know?

Reply:
I am actually 100% sure about that. There is no way we are storing that in any other way.

Reply:
Unless I am wrong, the only other option is to base it on the `locale`. That does mean a user can simply change their locale, see the banner, etc. They will still not be able to sign up unless they have an appropriate card.